packages/application-generic/src/services/analytics.service.ts

```diff
@@ -1,21 +1,38 @@
-import { UserEntity } from '@novu/dal';
-import { OrganizationEntity } from '@novu/dal';
 import Analytics from 'analytics-node';
 
 // Due to problematic analytics-node types, we need to use require
 // eslint-disable-next-line @typescript-eslint/naming-convention
 const AnalyticsClass = require('analytics-node');
 
+interface IOrganization {
+  name?: string | null;
+  createdAt?: string | null;
+}
+
+interface IUser {
+  firstName?: string | null;
+  lastName?: string | null;
+  email?: string | null;
+  profilePicture?: string | null;
+  createdAt?: string | null;
+}
```

Comment:
this service should be independent of the entity interfaces (dal), so it would be much easier to reuse it